### PR TITLE
Two more path for level up and a bug fix

### PR DESCRIPTION
**Pull request by bepehr**
When a buff like frost armor in mage expires and char is mounted bot will stock in a loop of cast buff again without dismount and its a bug , this code will fix this problem .

### CODE REVIEW

File: Libs/Goals/AdhocGoal.cs [hunk 1]
```diff
@@ -47,13 +47,13 @@ public override async Task PerformAction()
...
                 await this.stopMoving.Stop();
 
-                if (playerReader.PlayerBitValues.IsMounted)
-                {
-                    await wowProcess.Dismount();
-                }
                 await Task.Delay(1000);
             }
-
+            if (playerReader.PlayerBitValues.IsMounted)
```
Comment (julianperrott) [inline Libs/Goals/AdhocGoal.cs:52]: Use  StopBeforeCast property for this!

```
if (key.StopBeforeCast)
            {
                await this.stopMoving.Stop();

                if (playerReader.PlayerBitValues.IsMounted)
                {
                    await wowProcess.Dismount();
                }
                await Task.Delay(1000);
            }
```

**Comment by bepehr** [comment]
Oh you are right , please update default class json files so new users wont have this problem any more .
Thank you

File: Libs/Goals/AdhocGoal.cs (hunk 1 above)
Comment (julianperrott) [reply Libs/Goals/AdhocGoal.cs:52]: Can you undo the changes to this file and I can then accept the pull.

Comment (bepehr) [reply Libs/Goals/AdhocGoal.cs]: Done .

**Pull request merged**